### ISSUE TITLE
[Web] Another exception was thrown: Instance of 'minified:fi<void>'

### ISSUE THREAD

**Issue by hivesey** [flutter/flutter#71410]
I am on "1.24.0-10.2.pre"

This issue is specific to Web.

I have a simple scrollable list (child of scaffold). I get "Another exception was thrown: Instance of 'minified:fi<void>'" exception when I double click on the browser window to expand to fullscreen and vice versa. When I gradually resize the browser I do NOT get this exception.

After using the app for few minutes ... resizing window takes 5+ seconds to refresh (i am using mackbook pro)

**Comment by iapicca** [comment]
@hivesey can you provide a minimal reproducible code sample

**Comment by darshankawar** [comment]
@iapicca 
Please provide a complete minimal reproducible code sample as requested above that shows the issue you are facing.
Thanks.

**Comment by shankri** [comment]
Sorry for the delay.

Scrollbar(
   isAlwaysShown: true,
   ListView.builder(
          controller: _listScrollController,
          shrinkWrap: true,
          [children .... CheckboxListTile]
)

I get the exception when isAlwaysShown=true; NO exception when it is false

I switch back to "1.23.0-18.1.pre" because of this issue "https://github.com/flutter/flutter/issues/70948" in "1.24.0-10.2.pre"

So i see this exception in both "1.23.0-18.1.pre" and "1.24.0-10.2.pre"

**Comment by hivesey** [comment]
Sorry for the delay.

Scrollbar(
isAlwaysShown: true,
ListView.builder(
controller: _listScrollController,
shrinkWrap: true,
[children .... CheckboxListTile]
)

I get the exception when isAlwaysShown=true; NO exception when it is false

I switch back to "1.23.0-18.1.pre" because of this issue "#70948" in "1.24.0-10.2.pre"

So i see this exception in both "1.23.0-18.1.pre" and "1.24.0-10.2.pre"

Note: I see this exception even with a simple list tile as well

**Comment by darshankawar** [comment]
@hivesey 
We would need a [complete minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) that we can copy paste and run in IDE to see the issue you are facing.

**Comment by hivesey** [comment]
When creating a simple example I figured the issue is because I have a 'Key' for my Scaffold based on media query. I have this key because my app is reactive for different screen sizes on the Web. I had to have this key without which widgets are being rebuilt appropriately.

**Key('${MediaQuery.of(context).size.width} _ ${MediaQuery.of(context).size.height}')**

Now I remove this and am wrapping it with Orientation Builder and it seems to work fine.

I am closing this issue for now.
